src/store/index.js

```diff
@@ -24,12 +24,14 @@ const reducer = combineReducers({
   routing,
   browser: responsiveStateReducer,
 });
+/* eslint-disable no-underscore-dangle */
```

Comment:
According to eslint docs this rule is only a stylistic choice.  We can just apply it project wide (I don't think we have a use case for underscores anyway)